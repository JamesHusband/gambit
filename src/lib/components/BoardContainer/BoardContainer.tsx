export const BoardContainer = ({ children }: { children: React.ReactNode }) => (
  <main className="flex-1 flex items-center justify-center">
    <div className="w-full max-w-[700px]">{children}</div>
  </main>
);
