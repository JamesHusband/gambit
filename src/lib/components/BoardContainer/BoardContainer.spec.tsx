import { render, screen } from "@testing-library/react";
import { BoardContainer } from "./BoardContainer";

describe("BoardContainer", () => {
  it("should render with children", () => {
    const { container } = render(<BoardContainer>Test</BoardContainer>);
    expect(container).toMatchSnapshot();
    expect(screen.getByText("Test")).toBeInTheDocument();
  });
});
