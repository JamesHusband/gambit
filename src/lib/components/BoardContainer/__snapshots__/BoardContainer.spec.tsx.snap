// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`BoardContainer should render with children 1`] = `
<div>
  <main
    class="flex-1 flex items-center justify-center"
  >
    <div
      class="w-full max-w-[700px]"
    >
      Test
    </div>
  </main>
`;